test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -30,7 +30,7 @@ public CustomSearchParamTests(HttpIntegrationTestFixture fixture)
             _fixture = fixture;
         }
 
-        [SkippableFact]
+        [Fact(Skip = "Failing intermittently due to known issue")]
```

Comment:
@brendankowitz , is there an appropriate way to track that disablement of this test?  Perhaps a work item to enable it later?  Or, is there an open issue/bug that I can add to that requires that this test be re-enabled as part of its resolution?

Reply:
Thanks @shmartel, I added an issue here: https://github.com/microsoft/fhir-server/issues/1559, perhaps reference this in the comment